libsolidity/inlineasm/AsmData.h

```diff
@@ -64,6 +66,10 @@ struct VariableDeclaration { SourceLocation location; std::string name; std::sha
 struct Block { SourceLocation location; std::vector<Statement> statements; };
 /// Function definition ("function f(a, b) -> (d, e) { ... }")
 struct FunctionDefinition { SourceLocation location; std::string name; std::vector<std::string> arguments; std::vector<std::string> returns; Block body; };
+/// Switch case or default case
+struct Case { SourceLocation location; std::string name; Block body; };
```

Comment:
Yes, though we need to introduce some helpers for literals, i.e. to compare them.